Multisig/UI/Safe Management/Safe Settings/SafeSettingsContentView.swift

```diff
@@ -12,7 +12,7 @@ struct SafeSettingsContentView: View {
 
     @ObservedObject
     var safe: Safe
-
+    
```

Comment:
I think your Xcode is not configured to trim whitespaces 

<img width="912" alt="Screenshot 2020-05-14 at 11 15 18" src="https://user-images.githubusercontent.com/284056/81916569-7e660d00-95c3-11ea-9326-443509f01158.png">